hugegraph-api/src/main/java/org/apache/hugegraph/api/job/RebuildAPI.java

```diff
@@ -53,6 +53,7 @@ public class RebuildAPI extends API {
     @Status(Status.ACCEPTED)
     @Produces(APPLICATION_JSON_WITH_CHARSET)
     @RolesAllowed({"admin", "$owner=$graph $action=index_write"})
+    @RedirectFilter.RedirectMasterRole
```

Comment:
maybe ACCEPTED and RedirectMasterRole often appear simultaneously

Reply:
yes, if ACCEPTED appear, the RedirectMasterRole also appear